docs/scos/dev/setup/quickstart-guides-install-spryker/quickstart-guide-install-spryker-on-macos-and-linux.md

```diff
@@ -70,16 +70,24 @@ Update the hosts file using the command provided in the output of the previous s
 docker/sdk up
 ```
 
-You can now access your project using the following endpoints:
+Default demo shop configuration includes 3x stores *de*, *at*, and *us*.
 
+You can now access your project applications using the following endpoints:
+1. German Store
 * Storefront: `yves.de.spryker.local`
 * Back Office: `backoffice.de.spryker.local`
 
+2. Austrian Store
+* Storefront: `yves.at.spryker.local`
+* Back Office: `backoffice.at.spryker.local`
```

Comment:
Are these the only domains that get available?
I thing Jenkins, and other are also available.

@andriitserkovnyi could you please add other apps?

Reply:
are the other ones important for the "quick" guide? 

Another idea: Maybe just add the dashboard `spryker.local`  ?

Reply:
The Quick guide needs to explain what you get in the end.

>Another idea: Maybe just add the dashboard spryker.local ?
It's a good question to @a-sabaa :)

Reply:
So on dashboard that we open on `spryker.local` URL we will get a list of all available resources that we can use from the current setup . 
Including all yves, backoffice, mp, glue-storefront, glue-backend, glue for each store and services like queue and scheduler.

So mb we can just put an url here and mention that user can check all other available resources there

Reply:
I agree that the link to spryker.local would be the ideal solution. Shall I add it?

Reply:
Yep. With some description e.g. `You can now check all existing endpoints by opening spryker.local`